### PR TITLE
[FEATURE] Composer script for reading registered bundles

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

**Comment by oliverklee** [comment]
Still working on fixing the test failure …

**Comment by oliverklee** [comment]
The CI build is green now.

File: Classes/Composer/ScriptHandler.php [hunk 1]
```diff
@@ -151,4 +156,23 @@ private static function calculateMaximumPackageNameLength(array $modules): int
 
         return $maximumLength;
     }
+
+    /**
+     * @param Event $event
```
Comment (samtuke) [inline Classes/Composer/ScriptHandler.php]: Please can you add a short description of the functionality here?

File: Configuration/PHPMD/rules.xml [hunk 2]
```diff
@@ -42,7 +42,7 @@
     <!-- rules from the "naming" rule set -->
     <rule ref="rulesets/naming.xml/ShortVariable">
         <properties>
-            <property name="exceptions" value="id"/>
+            <property name="exceptions" value="id,lf"/>
```
Comment (samtuke) [inline Configuration/PHPMD/rules.xml]: Why are only these two short var names allowed? Other valid short names may foreseeably be added, and limiting to two seems arbitrary.

**Comment by samtuke** [comment]
From my side this is ready for merge. My comments above are not blockers. I'll wait for you to give feedback before proceeding and closing.

File: Configuration/PHPMD/rules.xml (hunk 2 above)
Comment (oliverklee) [reply Configuration/PHPMD/rules.xml]: Generally, short variable names tend to be not very expressive (and we want our code to be expressive). So I'd like to have very few case-by-case exceptions to the "no very short names" rule.

I could move this from the configuration file to annotations in the code comments, though. What do you think, @samtuke?

File: Classes/Composer/ScriptHandler.php (hunk 1 above)
Comment (oliverklee) [reply Classes/Composer/ScriptHandler.php]: Done.

**Comment by oliverklee** [comment]
All issues are resolved. I've repushed.

**Comment by samtuke** [comment]
Changes look good to me as-is. Merging!

**Pull request merged**